index.js

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import expressValidator from 'express-validator';
+import morgan from 'morgan';
+import routes from './server/routes/v1';
+
+const app = express();
+
+app.use(morgan('dev'));
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({
+  extended: false
+}));
+app.use(expressValidator());
+
+/** Solves error resulting from setting header when header is already sent */
+app.use((req, res, next) => {
+  const send = res.send;
+  let sent = false;
+  res.send = (data) => {
```

Comment:
***Code Quality: Critical***
why are you mutating hte content of res.send.. isn't this suppose to be called?